integration/apps/rspec/bin/setup

```diff
@@ -14,4 +14,10 @@ chdir APP_ROOT do
   puts "\n== Installing dependencies =="
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
+
+  # Create an non-trivial git repository, to augment test payload information.
+  system! 'git init'
+  system! 'GIT_AUTHOR_NAME="Test Author" GIT_AUTHOR_EMAIL=author@local.test ' \
+          'GIT_COMMITTER_NAME="Test Committer" GIT_COMMITTER_EMAIL=committer@local.test ' \
+          'git commit --allow-empty -m "Test message"'
 end
```

Comment:
This is slightly problematic when used with the current `docker-compose.yml`, which mounts the current directory directly into the container and thus means that this changes the host filesystem (and thus breaks working on this folder in the repo).

Perhaps create this as a subfolder?

Reply:
Addressed: .git stays only inside the container now, as I added a volume just for the .git directory.